src/rosdep2/platforms/pip.py

```diff
@@ -70,6 +84,41 @@ def get_pip_command():
     return None
 
 
+def externally_managed_installable():
+    """
+    PEP 668 enacted in Python 3.11 blocks pip from working in "externally
+    managed" envrionments such operating systems with included package
+    managers. If we're on Python 3.11 or greater, we need to check that pip
+    is configured to allow installing system-wide packages with the
+    flagrantly named "break system packages" config option or environment
+    variable.
+    """
+    if sys.version_info >= (3, 11):
```

Comment:
you could flip this to return early and remove one indent level from the rest of the function

```suggestion
    if sys.version_info < (3, 11):
        return True
```